src/components/Tab/Tab.js

```diff
@@ -28,14 +44,19 @@ const Tab = ({ onSelect, href, isSelected, children, description }) => {
 
 Tab.propTypes = {
   children: PropTypes.node.isRequired,
+  className: PropTypes.string,
   onSelect: PropTypes.func,
-  description: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  onClick: PropTypes.func,
```

Comment:
What is the `onClick` purpose? is `onSelect` not enough?

Reply:
I think`onClick` should be the main function to handle tabs.

Reply:
So remove onSelect ;d

Reply:
already did it ;)